lib/action_policy/lookup_chain.rb

```diff
@@ -114,7 +114,19 @@ def policy_class_name_for(record)
     SYMBOL_LOOKUP = ->(record, namespace: nil, **) {
       next unless record.is_a?(Symbol)
 
-      policy_name = "#{record.classify}Policy"
+      policy_name = "#{record.camelize}Policy"
+      if namespace.nil?
+        policy_name.safe_constantize
+      else
+        lookup_within_namespace(policy_name, namespace)
+      end
+    }
+
+    CLASSIFY_SYMBOL_LOOKUP = ->(record, namespace: nil, **) {
+      next unless record.is_a?(Symbol)
+      next unless record.to_s.respond_to?(:classify)
```

Comment:
Can we check for `respond_to?(:classify)` before building the chain, so we can skip this resolver altogether when it's missing? This way we can get rid of the `respond_to?` check and optimize the resolver.

Reply:
Excellent idea 👍  I'll update